dll/win32/shell32/shellrecyclebin/recyclebin_v5_enumerator.c

```diff
@@ -3,10 +3,11 @@
  * LICENSE:     GPL v2 - See COPYING in the top level directory
  * FILE:        lib/recyclebin/recyclebin_v5_enumerator.c
  * PURPOSE:     Enumerates contents of a MS Windows 2000/XP/2003 recyclebin
- * PROGRAMMERS: Copyright 2006-2007 Herv� Poussineau (hpoussin@reactos.org)
+ * PROGRAMMERS: Copyright 2006-2007 Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
Is the change here due to UTF-8 conversion? (if so, this is OK)

Reply:
Yeah, it's that, i rewrited the character as utf-8 reqrites it as some strange triangle